<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="Niki-PC" name="testdesarrollo.UserSpec" tests="5" time="5.358" timestamp="2015-10-22T01:05:09">
  <properties />
  <testcase classname="testdesarrollo.UserSpec" name="test name" time="4.097">
    <failure message="Condition not satisfied:&#xa;&#xa;usr.hasErrors() == !valido&#xa;|   |           |  ||&#xa;|   true        |  |true&#xa;|               |  false&#xa;|               false&#xa;testdesarrollo.User : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Condition not satisfied:

usr.hasErrors() == !valido
|   |           |  ||
|   true        |  |true
|               |  false
|               false
testdesarrollo.User : (unsaved)

	at testdesarrollo.UserSpec.test name(UserSpec.groovy:28)
</failure>
  </testcase>
  <testcase classname="testdesarrollo.UserSpec" name="test lastName" time="1.037" />
  <testcase classname="testdesarrollo.UserSpec" name="test AGE" time="0.047" />
  <testcase classname="testdesarrollo.UserSpec" name="test username" time="0.036" />
  <testcase classname="testdesarrollo.UserSpec" name="test password" time="0.045" />
  <system-out><![CDATA[--Output from test name--
--Output from test lastName--
--Output from test AGE--
--Output from test username--
--Output from test password--
]]></system-out>
  <system-err><![CDATA[--Output from test name--
--Output from test lastName--
--Output from test AGE--
--Output from test username--
--Output from test password--
]]></system-err>
</testsuite>
